vortexasdk/endpoints/voyages_timeseries_v2.py

```diff
@@ -29,19 +31,21 @@ class VoyagesTimeseriesV2(Search):
 
     # Arguments
 
-        metric: Used to build the value of the aggregation. Can be one of: `'voyage-count'`, `'tonne-miles-sum'`, `'distance-avg'`.
+        metric: Used to build the value of the aggregation. Can be one of: `'voyage-count'`, `'tonne-miles-sum'`, `'distance-avg'`, `'distance-voyage-avg'`.
 
-        breakdown_property: Defines the voyage property to aggregate upon. Can be one of: `'status'`, `'vessel-class-legacy'`, `'vessel-class-group'`, `'vessel-class-coarse'`, `'vessel-class-granular'`, `'origin-region'`, `'origin-shipping-region'`, `'origin-trading-region'`, `'origin-trading-sub-region'`, `'origin-trading-block'`, `'origin-country'`, `'origin-port'`, `'origin-terminal'`, `'destination-region'`, `'destination-shipping-region'`, `'destination-trading-region'`, `'destination-trading-sub-region'`, `'destination-trading-block'`, `'destination-country'`, `'destination-port'`, `'destination-terminal'`, `'product-group'`, `'product-group-product'`, `'product-category'`, `'product-grade'`, `'corporate-entity-cargo-trader'`, `'corporate-entity-charterer'`, `'corporate-entity-effective-controller'`, `'corporate-entity-vessel-owner'`, `'corporate-entity-time-charterer'`.
+        breakdown_property: Defines the voyage property to aggregate upon. Can be one of: `'status'`, `'vessel-class-legacy'`, `'vessel-class-group'`, `'vessel-class-coarse'`, `'vessel-class-granular'`, `'origin-region'`, `'origin-shipping-region'`, `'origin-trading-region'`, `'origin-trading-sub-region'`, `'origin-trading-block'`, `'origin-country'`, `'origin-port'`, `'origin-terminal'`, `'origin_port_or_sts_zone'`, `'destination-region'`, `'destination-shipping-region'`, `'destination-trading-region'`, `'destination-trading-sub-region'`, `'destination-trading-block'`, `'destination-country'`, `'destination-port'`, `'destination-terminal'`, `'destination_port_or_sts_zone'`, `'location-waypoint'`, `'product-group'`, `'product-group-product'`, `'product-category'`, `'product-grade'`, `'corporate-entity-cargo-trader'`, `'corporate-entity-charterer'`, `'corporate-entity-effective-controller'`, `'corporate-entity-vessel-owner'`, `'corporate-entity-time-charterer'`.
```

Comment:
Are some missing? https://vortexa.slack.com/archives/CD5MA408P/p1726580430912939?thread_ts=1726580034.870479&cid=CD5MA408P

Reply:
I added all the properties supported in the API, including the ones disabled from the UI. I also removed some the legacy vessel class.